app_customers/lib/pages/transfer/component/form_transfert.dart

```diff
@@ -445,3 +450,92 @@ class _AlertMessageContent extends StatelessWidget {
     );
   }
 }
+
+class _ForfeitView extends StatelessWidget {
+  const _ForfeitView();
+
+  @override
+  Widget build(BuildContext context) {
+    final controller = Get.find<TransfersController>();
+    final Forfeit? forfeit = controller.forfeit;
+    final localization = Get.find<AppInternationalization>();
+
+    if (forfeit == null) {
+      return SizedBox();
+    }
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          localization.forfeitSelected,
+        ),
+        SizedBox(
+          height: 10,
+        ),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Expanded(
+              child: OperatorIcon(
+                operatorType: forfeit.reference.key,
+              ),
+            ),
+            Expanded(
+              flex: 2,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Text(
+                    forfeit.name,
+                    style: TextStyle(
+                      fontWeight: FontWeight.w500,
+                    ),
+                  ),
+                  Text(
+                    localization.locale.languageCode == 'en'
```

Comment:
can we make value 'en' in enum in order to avoid to make that in some order files ?